File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rsp/filterCodes/model/GetFilterCodes.java
package org.ngbp.jsonrpc4jtestharness.rsp.filterCodes.model;

import java.util.List;

public class GetFilterCodes {
    public List<Filters> filters;
    public static class Filters{
        public Integer filterCode;
        public String expires;

    }
}

File: nextgen-middleware/src/main/java/com/nextgenbroadcast/mobile/middleware/rpc/filterCodes/model/GetFilterCodes.kt
package com.nextgenbroadcast.mobile.middleware.rpc.filterCodes.model

import com.nextgenbroadcast.mobile.middleware.rpc.RpcResponse

data class GetFilterCodes(
        var filters: MutableList<Filters?>? = null
) : RpcResponse()

data class Filters(
        var filterCode: Int? = null,
        var expires: String? = null
)
